Source/WebKit/UIProcess/DrawingAreaProxy.h

```diff
@@ -140,6 +141,8 @@ class DrawingAreaProxy : public IPC::MessageReceiver, public IPC::MessageSender
     virtual void addRemotePageDrawingAreaProxy(RemotePageDrawingAreaProxy&) { }
     virtual void removeRemotePageDrawingAreaProxy(RemotePageDrawingAreaProxy&) { }
 
+    virtual void remotePageProcessCrashed(WebCore::ProcessIdentifier) { }
```

Comment:
Please use the same "didTerminate" terminology used at the WebProcessProxy level.

Reply:
Will do in a follow-up PR.